src/main/java/com/google/sps/travelbud/CountryServlet.java

```diff
@@ -48,18 +57,31 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       }
     }
   }
-  public Country getCountry(List<Country> countries, String path) {
-    // adjust endpoint
-    String countryName = path.substring(1).toLowerCase();
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    long id = Long.parseLong(getParameter(request, "id", ""));
+    String name = getParameter(request, "name", "");
+    List<String> cultureDos = Arrays.asList(getParameter(request, "cultureDos", "").split(",", -1));
+    List<String> cultureDonts =
+        Arrays.asList(getParameter(request, "cultureDonts", "").split(",", -1));
```

Comment:
Interesting. This means there can't be any commas in the list values. It looks like this though instead of splitting on `,` it might be better to have the front end figure out how to dynamically add more values to the same `cultureDos` and `cultureDonts` keys so that you can use [`HttpServletRequest.getParameterValues()`](https://docs.oracle.com/javaee/7/api/javax/servlet/ServletRequest.html#getParameterValues-java.lang.String-) instead:
```java
    List<String> cultureDos = Arrays.asList(request.getParameterValues("cultureDos"));
    List<String> cultureDonts = Arrays.asList(request.getParameterValues("cultureDonts"));
```